File: examples/python/pagination/snapshot.py
from nightona import Nightona


def main():
    nightona = Nightona()

    result = nightona.snapshot.list(page=2, limit=10)
    for snapshot in result.items:
        print(f"{snapshot.name} ({snapshot.image_name})")


if __name__ == "__main__":
    main()

File: examples/ruby/pagination/snapshot.rb
# frozen_string_literal: true

require 'nightona'

nightona = Nightona::Nightona.new

result = nightona.snapshot.list(page: 2, limit: 10)
result.items.each do |snapshot|
  puts "#{snapshot.name} (#{snapshot.image_name})"
end
